src/vs/workbench/contrib/preferences/browser/preferencesWidgets.ts

```diff
@@ -149,7 +149,7 @@ export class FolderSettingsActionViewItem extends BaseActionViewItem {
 		const workspace = this.contextService.getWorkspace();
 		if (this._folder) {
 			this.labelElement.textContent = this._folder.name;
-			this.anchorElement.title = (await this.preferencesService.getEditableSettingsURI(ConfigurationTarget.WORKSPACE_FOLDER, this._folder.uri))?.fsPath || '';
+			this.anchorElement.title = this._folder.name;
```

Comment:
If this section is the only thing making the function async, then the function can now be converted to a sync function that returns `void`.